docs/internal-documentation/templates/specifications/tests/TestCaseIdentifierSpecification.md

```diff
@@ -0,0 +1,32 @@
+# Test Case Identifier Specification
+
+All test cases belong to one specific test level, and it gets its name from that
+test level's name. The following test levels are defined and available:
+
+* Address
+* Basic
+* Connectivity
+* Consitency
+* DNSSEC
+* Delegation
+* Nameserver
+* Syntax
+* Zone
+
+The test case identifier has the format: <Test Level name> + <Index>
```

Comment:
String between `< >` are not rendered by github, better escape them as `\<Index>`: \<Index>

Reply:
Corrected.